planning/scenario_planning/parking/astar_search/include/astar_search/astar_search.h

```diff
@@ -17,17 +17,18 @@
 #ifndef ASTER_PLANNER_H
 #define ASTER_PLANNER_H
 
+#include <cmath>
 #include <functional>
 #include <iostream>
 #include <queue>
 #include <string>
 #include <tuple>
 #include <vector>
```

Comment:
[Minor]: Since I can only comment on it on this file, I don't know if the vector include is needed (in `astar_search.cpp`) since `vector` is already included in the `astar_search.h`?

Reply:
It is not needed. I removed it.